PMMain/Midas/MidasGUI/MyoTranslationFilter.cpp

```diff
@@ -35,24 +37,32 @@ void MyoTranslationFilter::process()
         baseYaw = yaw;
     }
 
+    outputToSharedCommandData[DELTA_VOL] = 0;
```

Comment:
This name is pretty specific to volume. Couldn't the roll be used for other purposes as well? Perhaps the name should be a bit more general at this stage.